rules/falco_rules.yaml

```diff
@@ -852,7 +852,7 @@
                           gen_resolvconf., update-ca-certi, certbot, runsv,
                           qualys-cloud-ag, locales.postins, nomachine_binaries,
                           adclient, certutil, crlutil, pam-auth-update, parallels_insta,
-                          openshift-launc, update-rc.d)
+                          openshift-launc, update-rc.d, ufw, cloud-init)
```

Comment:
For ufw, could we limit the scope to both the ufw program as well as the directory to which it writes? With something like this:

```
- macro: ufw_writing_config
  condition: proc.name=ufw and fd.directory=/etc/ufw
```
And then add ufw_writing_config to the (long) list of exceptions in write_etc_common.

For cloud-init, it looks like your example involves writing below /root/.ssh, right? In which case you want to add it as an exception to the `Write below monitored dir` rule instead. Again, ideally the exception would tie together the program as well as the file/directory being written.

I have some other rule updates pending, so if you'd like I can merge this PR and then make those changes in my rule updates branch once I rebase.

Reply:
no problem. good to me. Thanks @mstemm!